custom_components/variable/device_tracker.py

```
@@ -342,6 +350,11 @@ def state_attributes(self) -> dict[str, StateType]:
             attr[ATTR_LOCATION_NAME] = self._attr_location_name
         return attr
 
+    @property
+    def device_info(self) -> DeviceInfo:
```

Ruff

Missing type annotation for `self` in method

(ANN101)